src/components/molecules/ChangeFiltersConfirmModal/ChangeFiltersConfirmModal.tsx

```diff
@@ -0,0 +1,34 @@
+import {useEffect} from 'react';
+
+import {Modal} from 'antd';
+
+import {ExclamationCircleOutlined} from '@ant-design/icons';
+
+import {useAppDispatch, useAppSelector} from '@redux/hooks';
+import {extendResourceFilter, uncheckAllResourceIds} from '@redux/reducers/main';
+
+const ChangeFiltersConfirmModal: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const filtersToBeChanged = useAppSelector(state => state.main.filtersToBeChanged);
+
+  useEffect(() => {
+    if (filtersToBeChanged) {
+      Modal.confirm({
+        title: 'I understand that changing the filters will uncheck all my selected resources.',
```

Comment:
two thoughts:
- just use 'Changing the filter will uncheck all selected resources'
- can we uncheck only those that will be filtered away? - i.e. 'Changing the filter will uncheck all hidden resources'

Reply:
I think that would be a nice thing to have, but this would mean enabling back the Resource Filters Panel, which currently is disabled. And maybe we should think of another way of showing the panel, because the toggle button is hidden when you have selected resources ( maybe with some hotkey for opening it ).

Reply:
hm... why would we have to enable the filter panel in this scenario? The workflow I'm considering is:
- I select all deployments
- I realize I only want to export those in a specific namespace so I filter on that namespace by clicking on it in the navigator (or source editor)
- I now have to reselect all deployments (not a big deal...) - instead of keeping the selection on those deployments that are still visible

Reply:
I agree, but I think that we should be consistent, allowing the user to change the namespace also from the filters panel maybe? He will be able to do that through other actions, so he might get confused if that filter is disabled in the panel.

Reply:
I know we discussed this yesterday with @tommywhitehead also - just don't remember what we concluded - i guess i'm proposing that you can continue to filter your objects after selecting - just that whenever your filter hides a selected resource - that resource automatically gets unselected... !?

Reply:
@tommywhitehead thoughts?

Reply:
@olensmar @devcatalin - For now I unchecked the hidden resources when applying a filter outside the Filters Pane. But will leave the Filters Pane disabled, because it looks that it's a little bit more complex than expected to enable it when there are selected resources. We can make an issue for that if it's necessary and solve it in another PR. wdyt?